### PR TITLE
[Documentation] Linking project overview dashboard to README

### PR DESCRIPTION
**Pull request by alexkarezin**
Adding a link to the README header. The linked dashboard describes overall repository structure. The goal is to help new developers explore project and understand architecture/dependencies. Direct link: https://sourcespy.com/github/openrocketopenrocket/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ OpenRocket is an Open Source project licensed under the [GNU GPL](https://www.gn
 
 Contributing
 ------------
+[![SourceSpy Dashboard](https://sourcespy.com/shield.svg)](https://sourcespy.com/github/openrocketopenrocket/)
```
Comment (itshanney) [inline README.md:41]: @alexkarezin Can you move this to be after the Travis build status above and remove the Build Status header?

Comment (alexkarezin) [reply README.md]: @itshanney changed as requested

**Comment by itshanney** [comment]
@teyrana LGTM for merge. Thanks for the contribution, @alexkarezin.

**Comment by wolsen** [comment]
@alexkarezin I see a lot of pull requests from you in recent days adding these links to READMEs of various projects. Can you state a bit more clearly about what sourcespy is (the name already throws me off), how it helps and why its beneficial to have here vs adding different and more relevant documentation to what we currently have?

Also, why is it most appropriate on the README for the top level project documentation rather than some developer specific documentation, etc? Granted, Github is for developers - but its also for users reporting issues here. I'm just trying to wrap my head around whether this is the right place for this and what the real value this provides?

Before we link to other 3rd party sites, I'd like to have an idea of what that 3rd party site does? Who's behind it? What data are you collecting from the visitors? Is this Open Source? What's the license for use? etc.

**Comment by alexkarezin** [comment]
@wolsen Abolutely - I'll be happy to clarify.

This dashboard is an honest attempt to help new developers familiarize themselves with the project faster. In my experience (20+ years in dev), before any attempts to contribute to the project it helps to get a 10000ft view of major components and tech. Once developer gets a grasp of technologies and scope they can decide where it is their cup of tea and start diving further into documentation. 

The dashboard is my personal project - I'm working on it for close to a year now. The service is completely automated (downloads project and analyses code, builds diagrams) hence requires no effort from project maintainers.

I believe it helps new contributors save time and make involvement decisions faster be that OpenRocket or any other project. Happy to answer any other questions or make adjustments to the dashboard content.

**Pull request merged**